index.csv

```diff
@@ -225,3 +225,5 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+domutils,BSD-2-Clause,1.5.1
+css-select,BSD-2-Clause,1.2.0
```

Comment:
v1.2.0 is declared as "BSD-like": https://unpkg.com/browse/css-select@1.2.0/package.json, however, the license file is unchanged between v2.1.0, where it's marked as BSD-2-Clause, and v1.0.0, and every version in between.

Thus, can you also add entries for v1.0.0, v1.1.0, v1.3.0-rc0, and v2.0.0?